jekyll/_cci2/server/operator/installing-server-behind-a-proxy.adoc

```diff
@@ -16,32 +16,47 @@ toc::[]
 
-[#configuring-your-proxy]
-=== Configuring your proxy
-Once you have installed server and accessed the management console, there are some fields that need to be completed in the configuration section, as shown in the screenshot below. These fields will not be automatically populated, so the same proxy and no-proxy addresses you supplied during installation will need to be supplied here. If your proxy requires authentication in the form of a username and password, check the *HTTP Proxy authenticated* option to add the credentials.
+In order to configure authentication, you will need to configure `proxy.http.auth.enabled` and `proxy.https.auth.enabled` as `"1"`. You will also need to configure the respective username and password for both HTTP and HTTPS.
 
-.CircleCI Server v4.x Configuring a Proxy
-image::proxy-settings.png[Screenshot showing settings available to configure a proxy]
+Finally, configure the `no_proxy` hosts and subnets. This should include localhost, your GitHub Enterprise host (optional), and the hostname of your CircleCI installation (see limitations below for an explanation).
+
+```yaml
+proxy:
+  enabled: "1"
+  http:
+    host: proxy.example.internal
+    port: "3128"
+    auth:
+      enabled: "1"
+      username: circleci
+      password: circleciforever
+  https:
+    host: proxy.example.internal
+    port: "3128"
+    auth:
+      enabled: "1"
+      username: circleci
+      password: circleciforever
+  no_proxy:
+    - localhost
+    - 127.0.0.1
+    - github.example.internal
+    - circleci.example.internal
+    - 10.0.0.0/16 # Subnets to exclude from the proxy (optional)
+```
 
 [#known-limitations]
 == Known limitations
 
+* The CircleCI hostname must be added to the no-proxy list because the following services: `output processor` and `vm-service`. The application and build-agent share a no-proxy list and are assumed to be behind the same firewall and therefore cannot have a proxy between them.
 * Some additional configuration is required to import orbs when installed behind a proxy. See https://circleci.com/docs/2.0/server-3-operator-orbs/#using-orbs-behind-a-proxy[Orbs on Server] docs for more information.
 * The JVM only accepts proxies that run over HTTP, not HTTPS, and therefore proxy URIs must be of the form `\http://user:password@host:port` rather than `\https://user:password@host:port`.
 * If your GitHub instance is running outside of the proxied environment (either GitHub.com or GitHub Enterprise), you must ensure that SSH traffic from our application (inside the Kubernetes cluster) and from our Nomad node can reach your instance without additional configuration. Our SSH agents do not respect proxy settings because it uses a different network protocol. If a proxy is the only way that any traffic can reach outside the proxied environment, this means it will block SSH traffic and your application will fail.
```

Comment:
This bullet point needs help. We cannot proxy ssh-keyscan but we can technically proxy SSH itself.